src/helm/proxy/models.py

```diff
@@ -866,6 +866,15 @@ def engine(self) -> str:
         "([paper](https://arxiv.org/pdf/2204.02311.pdf)).",
         tags=[TEXT_MODEL_TAG, LIMITED_FUNCTIONALITY_TEXT_MODEL_TAG],
     ),
+    # Nvidia
+    Model(
+        group="megatron",
+        creator_organization="Nvidia",
+        name="megatron/gpt2",
```

Comment:
I think it would be more appropriate to set group to "nvidia" and set the name to "nvidia/megatron-gpt2".

Reply:
Done.